ros/src/twist_controller/dbw_node.py

```diff
@@ -47,6 +47,9 @@ def __init__(self):
         max_lat_accel = rospy.get_param('~max_lat_accel', 3.)
         max_steer_angle = rospy.get_param('~max_steer_angle', 8.)
 
+        rospy.set_param('dbw_decel_limit', decel_limit)
```

Comment:
I'm not sure what you are doing here. Please correct me if I am wrong. So you are creating new parameters 'dbw_decel_limit' and 'dbw_accel_limit' and setting there default values to decel_limit and accel_limit (both obtained from dbw_sim.launch). They share the same values. Why do we have to create new parameter?

Reply:
I didn't know how to access those parameters from Dow_node. I just found, using `rosparam list`  that you can access those via `/dbw_node/...`. fixing the issue